src/hermes_claude_agent_sdk/parity/v4_live_packets.py

```
@@ -0,0 +1,399 @@
+            trials[path], packets[path] = trial, packet
+        scenario_receipt = dict(core, receipt_hash=scenario_hash, provider_accounting={"positive_calls": selected.turn_count, "denial_calls": 0, "recovery_calls": 0, "total_calls": selected.turn_count})
+        return {"scenario": selected.to_dict(), "scenario_receipt": scenario_receipt, "scenario_receipt_hash": scenario_hash, "paths": paths, "trials": trials, "packets": packets}
+    except V4LivePacketViolation:
+        raise
+    except Exception as exc:
+        raise V4LivePacketViolation("scenario packet binding failed closed") from exc
+build_v4_live_packet_bundle = build_v4_live_packets
+build_v4_scenario_packets = build_v4_live_packets
+build_live_packets = build_v4_live_packets
+LivePacketViolation = V4LivePacketViolation
+__all__ = ["LIVE_MAP_SHA256", "LivePacketViolation", "V4LivePacketViolation", "build_live_packets", "build_v4_live_packet_bundle", "build_v4_scenario_packets", "build_v4_live_packets"]
```

Ruff (0.16.3)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)